### ISSUE TITLE
cd not centering

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#1391]
At this page of the sample article:
https://pretextbook.org/examples/sample-article/html/section-20.html

there is `cd` content. I believe it is supposed to be centered for display, but it is flush-left. The HTML source uses `<pre class="code-block tex2jax_ignore">` which is identical to what the HTML is using there for the PTX `pre` content, so there seems to be nothing for CSS to distinguish them.

(Should `cd` be centered? if there are two blocks of `cd` separated by some regular `p` content, would it be nicer if they were left-aligned with each other? Maybe both indented significantly?)

**Comment by rbeezer** [comment]
Yes, this mis-match has been on my mind for some time.  I guess it would be the time to decide which we want, flush-left or centered (as a block, not line-by-line).  I guess I vote for centered, as in PDF output, and as I think the documentation says or implies.

If so, I think we need input from @davidfarmer about how to organize classes.  One new one for centered `cd` or two new ones for existing `pre` versus centered `cd`?  Or ...?

Thanks for bringing this one forward.

**Comment by rbeezer** [comment]
Largely for @davidfarmer 

We already have CSS classes on appropriate HTML elements:
* `code.code-inline`, from PTX `c` and friends (e.g. `url`)
* `pre.code-block`, from `cd` and `pre`

The `div.code-box` is separate device to position `program` and similar.

At 512bf5d956f01262bf245f8cf45d8607e0f3a54e a PTX `cd` now becomes a `pre.code-display`.  This new class should center its contents (as a block) and maybe the vertical separation could/should match that of `md` since its intra-paragraph application is entirely similar.

I think this will be fine with backward-compatibility when `code-display` gets rules.  And at the moment I don't even see any visual harm.  So change is live now.

Three applications in Section 20 of the sample article to test with:

https://pretextbook.org/examples/sample-article/html/section-20.html

Thanks, @Alex-Jordan , for pushing this to the top of the stack.
